### DISCUSSION TITLE
openssl 3.2.1 downloading 0 byte tar file

### DISCUSSION THREAD

**Discussion by girishr-github** [openssl/openssl#25122] [Q&A] [answered]
curl -sO https://www.openssl.org/source/openssl-3.2.1.tar.gz. - gives a 0byte file. please help.
0 Aug  7 14:01 openssl-3.2.1.tar.gz

**Comment by nhorman** [comment, marked as answer]
Add the curl -L option to follow redirects

**Comment by girishr-github** [comment]
Thanks it worked with added -L option.
